src/components/LHNOptionsList/types.ts

```diff
@@ -71,6 +71,9 @@ type OptionRowLHNDataProps = {
     /** Array of report actions for this report */
     reportActions: OnyxEntry<ReportActions>;
 
+    /** Array of report actions of IOU report related to this report last action */
```

Comment:
NAB: Could you please explain more about why this is needed, so that data displayed in the option item is updated when a new action is pushed, such as display the split amount owed?